### PR TITLE
Actually unpublish video instead of pausing

### PR DESCRIPTION
**Pull request by steelbrain**
Currently we do not actually unpublish the video when `setLocalVideoEnabled(false)` is called, we just stop it

The remote side is still getting that stream, it's just stuck. This PR makes it so that it's actually unpublished/republished so the remote side can handle it appropriately

Tested and worked on both iOS and Android

### CODE REVIEW

File: ios/RCTTWVideoModule.m [hunk 1]
```diff
@@ -188,7 +188,33 @@ - (void)addParticipantView:(TVIVideoView *)view sid:(NSString *)sid trackSid:(NS
 
 RCT_REMAP_METHOD(setLocalVideoEnabled, enabled:(BOOL)enabled setLocalVideoEnabledWithResolver:(RCTPromiseResolveBlock)resolve
                  rejecter:(RCTPromiseRejectBlock)reject) {
-  [self.localVideoTrack setEnabled:enabled];
+  if (self.localVideoTrack != nil) {
+    [self.localVideoTrack setEnabled:enabled];
+  }
+  if (self.room == nil) {
+    // No room to add/remove tracks from, so ignore.
+    return;
```
Comment (slycoder) [inline ios/RCTTWVideoModule.m:196]: You'll want to make sure to either call resolve or reject otherwise any JS promises will not progress.

Comment (steelbrain) [reply ios/RCTTWVideoModule.m]: Fixed, thank you

**Comment by steelbrain** [comment]
There was an iOS specific oversight, where it would not enable video on new calls if it had been disabled in last call. Fixed now

**Comment by slycoder** [comment]
Thank you so much @steelbrain !

**Comment by James3432** [comment]
Hi @steelbrain 
Thanks for working on this!  However, this PR seems to have caused an issue which prevents the local video from reappearing again when setLocalVideoEnabled is toggled off and back on.  The stream is still sent to other Twilio clients, it just doesn't render again locally.

I've only tested this on iOS so far. Could you let me know if you're seeing a similar issue? If not I'll provide more details so we can investigate further. Thanks!

**Pull request merged**